### PR TITLE
[16.0][MIG] tracking manager

### PR DESCRIPTION
**Pull request by micheledic**

### CODE REVIEW

File: tracking_manager/tests/test_tracking_manager.py [hunk 1]
```diff
@@ -135,22 +135,19 @@ def test_m2m_multi_line(self):
...
 
     def test_o2m_write_indirectly(self):
         self.partner.write(
             {
                 "bank_ids": [(1, self.partner.bank_ids[0].id, {"acc_number": "123"})],
             }
         )
-        self.assertEqual(len(self.messages), 1)
-        self.assertIn("Change", self.messages.body)
+        self.assertEqual(len(self.messages), 2)
```
Comment (thomaspaulb) [inline tracking_manager/tests/test_tracking_manager.py:150]: Why suddenly 2 instead of 1?

File: tracking_manager/tests/test_tracking_manager.py [hunk 2]
```diff
@@ -163,7 +160,7 @@ def test_o2m_write_indirectly_on_not_tracked_fields(self):
                 "bank_ids": [(1, self.partner.bank_ids[0].id, {"acc_number": "123"})],
             }
         )
-        self.assertEqual(len(self.messages), 0)
+        self.assertEqual(len(self.messages), 1)
```
Comment (thomaspaulb) [inline tracking_manager/tests/test_tracking_manager.py:163]: Why 1 instead of 0?

Comment (micheledic) [reply tracking_manager/tests/test_tracking_manager.py:163]: I tried with a clean installation of odoo and there Is already tracking enabled on those one2many.
If i have not this module installed, a note Is written with the changed value  log

**Review by gurneyalex — APPROVED** [review]

**Comment by thomaspaulb** [comment]
@micheledic could you rebase and solve conflicts

**Comment by CRogos** [comment]
This PR can be canceled. Already migrated with https://github.com/OCA/server-tools/pull/2789 . Sorry did not see this PR before starting my migration.

**Pull request closed without merging**